java/find-service-area-interactive/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity
+            android:name="com.esri.arcgisruntime.sample.findserviceareainteractive.MainActivity"
```

Comment:
If the directory structure matches the package name, this should still be `.MainActivity`